cypress/utils/mockErrorResponse.ts

```diff
@@ -16,7 +16,7 @@ export const mockErrorResponse = ({
   errorCode = "INTERNAL_SERVER_ERROR",
   path = "i am a path", // the name of the query
 }: Args) => {
-  cy.route2("/graphql/query", (req) => {
+  cy.intercept("/graphql/query", (req) => {
```

Comment:
I forgot about this method and i see that we use it in a few places. Could we update everywhere we use `cy.intercept` to use this method. Could we also update this method to behave like the intercepts defined in `configure_patch.ts`? So we have more control over what requests get intercepted.